rllib/evaluation/postprocessing.py

```diff
@@ -205,7 +205,7 @@ def compute_gae_for_sample_batch(
             # will not be needed but takes time to compute.
             input_dict = policy._lazy_tensor_dict(input_dict)
             fwd_out = policy.model.forward_exploration(input_dict)
-            last_r = fwd_out[SampleBatch.VF_PREDS]
+            last_r = fwd_out[SampleBatch.VF_PREDS][0]
```

Comment:
wait, does this work for both torch and tf? If you are making this change here, what is the change on the other side to make torch not break?

Reply:
oh you know in hindsight this should actually be fwd_out[SampleBatch.VF_PREDS][-1] in order to be correct. 

last r is supposed to be the value estimation of the last observation.

I don't know why this didn't break the torch one, but the torch tests seem to be passing.

Reply:
changed to -1